nhs/data/handling.py

```diff
@@ -111,9 +111,9 @@ def read_spreadsheets(
 def standardize_names(
     df_dict: dict[str, pl.LazyFrame],
     census_metadata: pl.LazyFrame,
-    census_code_col: str,
-    abbreviation_column_name: str,
-    long_column_name: str,
+    census_code_col: str = "DataPackfile",
+    abbreviation_column_name: str = "Long",
+    long_column_name: str = "Short",
 ) -> dict[str, pl.LazyFrame | None]:
```

Comment:
I'm adding in preset settings since the only file we'll be using is the metadata CSV file with these column names. This both allow user to change the column names if they wish without requiring them to enter the column names explicitly.

Reply:
Also I'm removing `| None`, the function need to require user to put in something, it must impose restriction on how the user can use it.